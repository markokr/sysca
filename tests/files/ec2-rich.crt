-----BEGIN CERTIFICATE-----
MIIJQDCCCOagAwIBAgICAP8wCgYIKoZIzj0EAwIwggI4MRMwEQYDVQQDDApjb21t
b25OYW1lMQswCQYDVQQGEwJVUzERMA8GA1UEBwwITG9jYWxpdHkxDjAMBgNVBAgM
BVN0YXRlMQwwCgYDVQQKDANvcmcxETAPBgNVBAsMCHVuaXQxb2YyMREwDwYDVQQL
DAh1bml0Mm9mMjEYMBYGA1UECQwPc3RyZWV0QWRkcl8xb2YzMRIwEAYDVQQJDAlG
b296YSAxLzIxEjAQBgNVBAkMCUzDpGZ0IDRcYjEOMAwGA1UEEAwFcG9zdDExDjAM
BgNVBBAMBXBvc3QyMQ8wDQYDVQQPDAZiemNhdDExDzANBgNVBA8MBmJ6Y2F0MjEU
MBIGCgmSJomT8ixkARkWBGRvbTExFDASBgoJkiaJk/IsZAEZFgRkb20yMRAwDgYD
VQQEDAdzdXJuYW1lMRIwEAYDVQQqDAlnaXZlbk5hbWUxDjAMBgNVBAwMBXRpdGxl
MQ8wDQYDVQRBDAZwc2V1ZG8xDTALBgNVBCwMBGdlbnExDDAKBgNVBC4TA2RuUTET
MBEGCgmSJomT8ixkAQEMA3VpZDEQMA4GA1UELQwHeDUwMHVpZDEWMBQGCSqGSIb3
DQEJARYHZUBtYS5pbDEQMA4GA1UEBRMHeHNlcmlhbDERMA8GA1UEEQwIcG9zdENv
ZGUxEzARBgsrBgEEAYI3PAIBAxMCVVMxFzAVBgsrBgEEAYI3PAIBAQwGakxvY2Fs
MRcwFQYLKwYBBAGCNzwCAQIMBmpTdGF0ZTAgFw05NTEyMjQwMDAwMDBaGA8yMTk1
MTIyNDAwMDAwMFowggI4MRMwEQYDVQQDDApjb21tb25OYW1lMQswCQYDVQQGEwJV
UzERMA8GA1UEBwwITG9jYWxpdHkxDjAMBgNVBAgMBVN0YXRlMQwwCgYDVQQKDANv
cmcxETAPBgNVBAsMCHVuaXQxb2YyMREwDwYDVQQLDAh1bml0Mm9mMjEYMBYGA1UE
CQwPc3RyZWV0QWRkcl8xb2YzMRIwEAYDVQQJDAlGb296YSAxLzIxEjAQBgNVBAkM
CUzDpGZ0IDRcYjEOMAwGA1UEEAwFcG9zdDExDjAMBgNVBBAMBXBvc3QyMQ8wDQYD
VQQPDAZiemNhdDExDzANBgNVBA8MBmJ6Y2F0MjEUMBIGCgmSJomT8ixkARkWBGRv
bTExFDASBgoJkiaJk/IsZAEZFgRkb20yMRAwDgYDVQQEDAdzdXJuYW1lMRIwEAYD
VQQqDAlnaXZlbk5hbWUxDjAMBgNVBAwMBXRpdGxlMQ8wDQYDVQRBDAZwc2V1ZG8x
DTALBgNVBCwMBGdlbnExDDAKBgNVBC4TA2RuUTETMBEGCgmSJomT8ixkAQEMA3Vp
ZDEQMA4GA1UELQwHeDUwMHVpZDEWMBQGCSqGSIb3DQEJARYHZUBtYS5pbDEQMA4G
A1UEBRMHeHNlcmlhbDERMA8GA1UEEQwIcG9zdENvZGUxEzARBgsrBgEEAYI3PAIB
AxMCVVMxFzAVBgsrBgEEAYI3PAIBAQwGakxvY2FsMRcwFQYLKwYBBAGCNzwCAQIM
BmpTdGF0ZTBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABBNLisQDzINdrxbkhDim
CKPxo+kpCQZ7LDuPW8wY6NmHgLfeXQinlSXvK11OlfC9+LqDXk7fTHLSCnhH3C9V
712jggPYMIID1DASBgNVHRMBAf8ECDAGAQH/AgEAMA4GA1UdDwEB/wQEAwIB/jBO
BgNVHSUBAf8ERDBCBggrBgEFBQcDAgYIKwYBBQUHAwEGCCsGAQUFBwMDBggrBgEF
BQcDBAYIKwYBBQUHAwgGCCsGAQUFBwMJBgRVHSUAMIGMBgNVHREEgYQwgYGCDSou
ZXhhbXBsZS5jb22HBH8AAAGHEIAAAAAAAAAAAAAAAAAAAAGHCAoAAAD/AAAAhyD/
gAAAAAAAAAAAAAAAAAAA//////////8AAAAAAAAAAIYNaHR0cDovL2xvY2FsL4EK
bWVAcXFxLmNvbaQRMA8xDTALBgNVBAwMBEzEmXQwQQYDVR0fBDowODA2oDSgMoYX
aHR0cDovL2NybDAuZXhhbXBsZS5jb22GF2h0dHA6Ly9jcmwxLmV4YW1wbGUuY29t
MIGzBggrBgEFBQcBAQSBpjCBozAkBggrBgEFBQcwAYYYaHR0cDovL29jc3AwLmV4
YW1wbGUuY29tMCQGCCsGAQUFBzABhhhodHRwOi8vb2NzcDEuZXhhbXBsZS5jb20w
KQYIKwYBBQUHMAKGHWh0dHA6Ly9mdHAuZXhhbXBsZS5jb20vY2EuY3J0MCoGCCsG
AQUFBzAChh5odHRwOi8vZnRwMi5leGFtcGxlLmNvbS9jYS5jcnQwDwYJKwYBBQUH
MAEFBAIFADAUBggrBgEFBQcBGAQIMAYCAQUCAREwDQYDVR02AQH/BAMCAQUwEgYD
VR0kAQH/BAgwBoABA4EBAjCBvAYDVR0gBIG0MIGxMDQGAyoDBDAtMBUGCCsGAQUF
BwIBFglsb2NhbGhvc3QwFAYIKwYBBQUHAgEWCGludGVybmFsMHEGA1MEBTBqMCUG
CCsGAQUFBwICMBkwEAwDb3JnMAkCAQECAQICAQMMBXdvcmRzMBIGCCsGAQUFBwIC
MAYMBG1vcmUwFgYIKwYBBQUHAgIwCjAIDARvbmx5MAAwFQYIKwYBBQUHAgIwCTAH
DAAwAwIBATAGBgRVHSAAMB0GA1UdDgQWBBRKNO71jW+wiu552gh/cJSxiG7DxzAf
BgNVHSMEGDAWgBRKNO71jW+wiu552gh/cJSxiG7DxzCBjAYDVR0SBIGEMIGBgg0q
LmV4YW1wbGUuY29thwR/AAABhxCAAAAAAAAAAAAAAAAAAAABhwgKAAAA/wAAAIcg
/4AAAAAAAAAAAAAAAAAAAP//////////AAAAAAAAAACGDWh0dHA6Ly9sb2NhbC+B
Cm1lQHFxcS5jb22kETAPMQ0wCwYDVQQMDARMxJl0MAoGCCqGSM49BAMCA0gAMEUC
IEA0/pL0OWumH824VqKMN8VohJzRGMuFxm7Acaz/3E1sAiEAmlmyht+dl8J966kL
mJFm6MwXaCaKuxx76nahkmFb8/Q=
-----END CERTIFICATE-----
