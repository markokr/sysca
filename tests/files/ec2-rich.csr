-----BEGIN CERTIFICATE REQUEST-----
MIIGEzCCBboCAQAwggI4MRMwEQYDVQQDDApjb21tb25OYW1lMQswCQYDVQQGEwJV
UzERMA8GA1UEBwwITG9jYWxpdHkxDjAMBgNVBAgMBVN0YXRlMQwwCgYDVQQKDANv
cmcxETAPBgNVBAsMCHVuaXQxb2YyMREwDwYDVQQLDAh1bml0Mm9mMjEYMBYGA1UE
CQwPc3RyZWV0QWRkcl8xb2YzMRIwEAYDVQQJDAlGb296YSAxLzIxEjAQBgNVBAkM
CUzDpGZ0IDRcYjEOMAwGA1UEEAwFcG9zdDExDjAMBgNVBBAMBXBvc3QyMQ8wDQYD
VQQPDAZiemNhdDExDzANBgNVBA8MBmJ6Y2F0MjEUMBIGCgmSJomT8ixkARkWBGRv
bTExFDASBgoJkiaJk/IsZAEZFgRkb20yMRAwDgYDVQQEDAdzdXJuYW1lMRIwEAYD
VQQqDAlnaXZlbk5hbWUxDjAMBgNVBAwMBXRpdGxlMQ8wDQYDVQRBDAZwc2V1ZG8x
DTALBgNVBCwMBGdlbnExDDAKBgNVBC4TA2RuUTETMBEGCgmSJomT8ixkAQEMA3Vp
ZDEQMA4GA1UELQwHeDUwMHVpZDEWMBQGCSqGSIb3DQEJARYHZUBtYS5pbDEQMA4G
A1UEBRMHeHNlcmlhbDERMA8GA1UEEQwIcG9zdENvZGUxEzARBgsrBgEEAYI3PAIB
AxMCVVMxFzAVBgsrBgEEAYI3PAIBAQwGakxvY2FsMRcwFQYLKwYBBAGCNzwCAQIM
BmpTdGF0ZTBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABBNLisQDzINdrxbkhDim
CKPxo+kpCQZ7LDuPW8wY6NmHgLfeXQinlSXvK11OlfC9+LqDXk7fTHLSCnhH3C9V
712gggMcMIIDGAYJKoZIhvcNAQkOMYIDCTCCAwUwEgYDVR0TAQH/BAgwBgEB/wIB
ADAOBgNVHQ8BAf8EBAMCAf4wTgYDVR0lAQH/BEQwQgYIKwYBBQUHAwIGCCsGAQUF
BwMBBggrBgEFBQcDAwYIKwYBBQUHAwQGCCsGAQUFBwMIBggrBgEFBQcDCQYEVR0l
ADCBjAYDVR0RBIGEMIGBgg0qLmV4YW1wbGUuY29thwR/AAABhxCAAAAAAAAAAAAA
AAAAAAABhwgKAAAA/wAAAIcg/4AAAAAAAAAAAAAAAAAAAP//////////AAAAAAAA
AACGDWh0dHA6Ly9sb2NhbC+BCm1lQHFxcS5jb22kETAPMQ0wCwYDVQQMDARMxJl0
MEEGA1UdHwQ6MDgwNqA0oDKGF2h0dHA6Ly9jcmwwLmV4YW1wbGUuY29thhdodHRw
Oi8vY3JsMS5leGFtcGxlLmNvbTCBswYIKwYBBQUHAQEEgaYwgaMwJAYIKwYBBQUH
MAGGGGh0dHA6Ly9vY3NwMC5leGFtcGxlLmNvbTAkBggrBgEFBQcwAYYYaHR0cDov
L29jc3AxLmV4YW1wbGUuY29tMCkGCCsGAQUFBzAChh1odHRwOi8vZnRwLmV4YW1w
bGUuY29tL2NhLmNydDAqBggrBgEFBQcwAoYeaHR0cDovL2Z0cDIuZXhhbXBsZS5j
b20vY2EuY3J0MA8GCSsGAQUFBzABBQQCBQAwFAYIKwYBBQUHARgECDAGAgEFAgER
MA0GA1UdNgEB/wQDAgEFMBIGA1UdJAEB/wQIMAaAAQOBAQIwgbwGA1UdIASBtDCB
sTA0BgMqAwQwLTAVBggrBgEFBQcCARYJbG9jYWxob3N0MBQGCCsGAQUFBwIBFghp
bnRlcm5hbDBxBgNTBAUwajAlBggrBgEFBQcCAjAZMBAMA29yZzAJAgEBAgECAgED
DAV3b3JkczASBggrBgEFBQcCAjAGDARtb3JlMBYGCCsGAQUFBwICMAowCAwEb25s
eTAAMBUGCCsGAQUFBwICMAkwBwwAMAMCAQEwBgYEVR0gADAKBggqhkjOPQQDAgNH
ADBEAiBzJjSB7cYYySBETxzK278JqMCDvV3SFhu41BW4ww7zmAIgJ5SeMVYfvU+x
Kyn/9/DEpDczLNMIc896tvrirS4O85M=
-----END CERTIFICATE REQUEST-----
