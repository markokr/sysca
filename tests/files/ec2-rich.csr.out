Public key: ec:secp256r1
Subject: CN = commonName, C = US, L = Locality, ST = State, O = org, OU = unit1of2, OU = unit2of2, STREET = streetAddr_1of3, STREET = Fooza 1/2, STREET = Läft 4\\b, PA = post1, PA = post2, BC = bzcat1, BC = bzcat2, DC = dom1, DC = dom2, SN = surname, GN = givenName, T = title, P = pseudo, GQ = genq, DQ = dnQ, UID = uid, XUID = x500uid, EMAIL = e@ma.il, SERIAL = xserial, PC = postCode, JC = US, JL = jLocal, JST = jState
Subject Alternative Name:
  dns:*.example.com
  ip:127.0.0.1
  ip:8000::1
  net:10.0.0.0/8
  net:ff80::/64
  uri:http://local/
  email:me@qqq.com
  dn:/T=Lęt/
CA: True
Path Length: 0
Usage: digital_signature, content_commitment, key_encipherment, data_encipherment, key_agreement, key_cert_sign, crl_sign, client, server, code, email, time, ocsp, any
Issuer URLs:
  http://ftp.example.com/ca.crt
  http://ftp2.example.com/ca.crt
OCSP URLs:
  http://ocsp0.example.com
  http://ocsp1.example.com
CRL URLs:
  http://crl0.example.com
  http://crl1.example.com
OCSP NoCheck: True
TLS Features:
  status_request
  status_request_v2
Inhibit ANY policy: skip_certs=5
Policy Constraint - Require Explicit Policy: 3
Policy Constraint - Inhibit Policy Mapping: 2
Certificate Policies:
  1.2.3.4:|P=localhost|,|P=internal|
  2.3.4.5:|O=org|N=1:2:3|T=words|,|T=more|,|O=only|,|N=1|
  2.5.29.32.0
